models/discordactions.js

```diff
@@ -87,9 +90,37 @@ const addGroupRoleToMember = async (roleData) => {
   }
 };
 
+/**
+ *
+ * @param userDiscordId { String }: DiscordId of the user
+ * @returns {Promise<String>}
+ */
+const updateDiscordImageForVerification = async (userDiscordId) => {
+  try {
+    const discordAvatarUrl = await generateDiscordProfileImageUrl(userDiscordId);
+    const verificationDataSnapshot = await photoVerificationModel
+      .where("discordId", "==", userDiscordId)
+      .limit(1)
+      .get();
+    const unverifiedUserDiscordImage = {
+      discord: { url: discordAvatarUrl, approved: false, date: admin.firestore.Timestamp.fromDate(new Date()) },
```

Comment:
We will always not be using firebase. we can use a more acceptable unit to store timestamp such as UTC time stamp or EPOCH timestamp.

Reply:
I followed the convention that is used all across other models for date-time.